### ISSUE TITLE
Error while decrypting and verifying M5 subTlv

### ISSUE THREAD

**Issue by MaciejKuzniar** [homebridge/homebridge#2999]
**Describe The Bug:**
I can pair iPhone 11 neither with IOS 15.0.2 or 14.8 with Homebridge.

[**Logs:**]()
HAP-NodeJS:EventedHTTPServer [::ffff:192.168.100.15] New connection from client on interface eth0 (::ffff:192.168.100.4) +18s
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] Internal HTTP server listening on 127.0.0.1:41899 +0ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] Internal HTTP socket connected. HAPConnection now fully set up! +21ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP request: /pair-setup +16ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] HAP Request: POST /pair-setup +0ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] Pair step 1/5 +11ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP Response is finished +1s
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP request: /pair-setup +147ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] HAP Request: POST /pair-setup +1s
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] Pair step 2/5 +2ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP Response is finished +931ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP request: /pair-setup +17ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] HAP Request: POST /pair-setup +947ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] Pair step 3/5 +2ms
  HAP-NodeJS:HAPServer [CC:22:3D:E3:CE:30] **Error while decrypting and verifying M5 subTlv: +5ms**
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP Response is finished +11ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] Client connection closed +14ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP connection was closed +3ms
  HAP-NodeJS:EventedHTTPServer:Connection [::ffff:192.168.100.15] HTTP server was closed +2ms

Tracing network traffic shows 3 attempts to connect \:
POST /pair-setup HTTP/1.1..Host: Homebridge\0324241\03252D2._hap._tcp.local [...]
HTTP/1.1 200 OK..Content-Type: application/pairing+tlv8
POST /pair-setup HTTP/1.1..Host: Homebridge\0324241\03252D2._hap._tcp.local
HTTP/1.1 200 OK..Content-Type: application/pairing+tlv8
POST /pair-setup HTTP/1.1..Host: Homebridge\0324241\03252D2
HTTP/1.1 200 OK..Content-Type: application/pairing+tlv8

and finally iPhone shows message cannot add accessory.

**Homebridge Config:**

```json
"bridge": {
        "name": "Homebridge 4241",
        "username": "XXX",
        "port": XXX,
        "pin": "XXX"
    },
    "accessories": [
        {
            "type": "switch",
            "name": "XXX",
            "url": "mqtt://127.0.0.1:1883",
            "mqttOptions": {
                "keepalive": 60
            },
            "mqttPubOptions": {
                "retain": false
            },
            "topics": {
                "getOnline": "supla/XXX/devices/6/channels/15/state/connected",
                "getOn": "supla/XXX/devices/6/channels/15/state/on",
                "setOn": "supla/XXX/devices/6/channels/15/set/on"
            },
            "confirmationIndicateOffline": true,
            "accessory": "mqttthing"
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": XXX,
            "platform": "config"
        }
    ]
}
```

**Environment:**

* **Node.js Version**: 17.0.1
* **NPM Version**: 8.1.0
* **Homebridge Version**: 1.3.5
* **Operating System**: Raspbian 
* **Process Supervisor**: Systemd

**Comment by hehongbo** [comment]
This problem seems related to node 17.
Downgraded to node 16 by switching from `FROM node:alpine` to `FROM node:16-alpine` in my Dockerfile and the problem seems get resolved.

Node.js 17.0.1 (I believe 17.0.0 won't work either)
Homebridge 1.3.5
Linux kernel 5.10.77

**Comment by hehongbo** [comment]
bump.

Can confirm that Node 17.2 is also broken.

**Comment by Supereg** [comment]
A fix for this is currently live as beta release `1.3.9-beta.0` which can be installed from the UI.

**Comment by Supereg** [comment]
This issue should be fixed with release 1.3.9
